fluentgen/src/test/java/com/azure/autorest/fluent/FluentGenTests.java

```diff
@@ -29,4 +29,14 @@ public void testProcess() {
         FluentJavaPackage javaPackage = fluentgenAccessor.handleTemplate(client);
         FluentClient fluentClient = fluentgenAccessor.handleFluentLite(codeModel, client, javaPackage);
     }
+
+    @Test
+    @Disabled
+    public void testLiveTestGen(){
+        CodeModel codeModel = TestUtils.loadCodeModel(fluentgenAccessor, "code-model-fluentnamer-signalr.yaml");
+        Client client = FluentStatic.getClient();
+        FluentJavaPackage javaPackage = fluentgenAccessor.handleTemplate(client);
+        FluentClient fluentClient = fluentgenAccessor.handleFluentLite(codeModel, client, javaPackage);
```

Comment:
I expect to have more tests here?

You can log and do this in next PR.

Reply:
Logged [here](https://github.com/Azure/autorest.java/issues/1383).